AmplifyPlugins/Auth/AWSAuthPlugin/Operations/AWSAuthConfirmSignInOperation.swift

```diff
@@ -0,0 +1,68 @@
+//
+// Copyright 2018-2020 Amazon.com,
+// Inc. or its affiliates. All Rights Reserved.
+//
+// SPDX-License-Identifier: Apache-2.0
+//
+
+import Foundation
+
+import Amplify
+
+public class AWSAuthConfirmSignInOperation: AmplifyOperation<AuthConfirmSignInRequest,
+    Void,
+    AuthSignInResult,
+    AmplifyAuthError>,
+AuthConfirmSignInOperation {
+
+    let authenticationProvider: AuthenticationProviderBehavior
+
+    init(_ request: AuthConfirmSignInRequest,
+         authenticationProvider: AuthenticationProviderBehavior,
+         listener: EventListener?) {
+
+        self.authenticationProvider = authenticationProvider
+        super.init(categoryType: .auth,
+                   eventName: HubPayload.EventName.Auth.confirmSignIn,
+                   request: request,
+                   listener: listener)
+    }
+
+    override public func main() {
+        if isCancelled {
+            finish()
+            return
+        }
+
+        if let validationError = request.hasError() {
+            dispatch(validationError)
+            finish()
+            return
+        }
+
+        authenticationProvider.confirmSignIn(request: request) {[weak self]  result in
```

Comment:
missing space in front of `[weak self]` and extra space after

Reply:
Fixed